_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-dinky
```

Comment:
Why exactly is this necessary?

Reply:
Save to remove this line it this is github template.

Compare local git to 
https://git.linuxtv.org/dtv-scan-tables.git/